### PR TITLE
[JAX FE] Add support for conv_general_dilated.

### PR DESCRIPTION
**Pull request by AsakusaRinne**
### Details:
 - *Add the translation for conv_general_dilated*
 - *Add corresponding tests*
 - *Add an environment variable in the workflow to skip the gpu check when getting jaxpr for conv.*

### Tickets:
 - *CVS-145576*

### CODE REVIEW

**Comment by AsakusaRinne** [comment]
@mvafin Could you please help to review this PR? :)

cc @rkazants

File: src/plugins/intel_gpu/thirdparty/onednn_gpu
Comment (mvafin) [inline src/plugins/intel_gpu/thirdparty/onednn_gpu:1]: Why this is changed? Please revert

File: src/bindings/python/src/openvino/frontend/jax/utils.py [hunk 1]
```diff
@@ -108,14 +109,27 @@ def ivalue_to_constant(ivalue, shared_memory=True):
     '''
     Convert a python object to an openvino constant.
     '''
+    ivalue = filter_ivalue(ivalue)
     ov_type = get_type_from_py_type(ivalue)
     if ov_type.is_static():
         return op.Constant(ov_type, Shape([]), [ivalue]).outputs()
-
+
```
Comment (mvafin) [inline src/bindings/python/src/openvino/frontend/jax/utils.py:116]:
```suggestion

```

**Comment by mvafin** [comment]
Is this PR really needed? I see all the relevant changes in #25292

**Pull request closed without merging**